internal/pkg/fflag/flipt/client.go

```diff
@@ -0,0 +1,85 @@
+package flipt
```

Comment:
@Dotunj This library isn't doing too much; we shouldn't be logging here. The client code should handle logging while this library returns errors.